src/catalog/src/information_schema/runtime_metrics.rs

```diff
@@ -170,18 +184,19 @@ impl InformationSchemaMetricsBuilder {
                     .join(", "),
                 // Safety: always has a sample
                 ts.samples[0].value,
+                // The peer column is always `None` for standalone
+                None,
+                "standalone",
```

Comment:
```suggestion
                "STANDALONE",
```
Because here's `STANDALONE`, I think we should unify the naming. https://github.com/GreptimeTeam/greptimedb/blob/97babb0457ec605681d6157fd59c0e694bdc3002/src/common/meta/src/cluster.rs#L119